### DISCUSSION TITLE
Doco typo in 'deployment'

### DISCUSSION THREAD

**Discussion by omerzubair** [Azure/ipam#215] [General]
just a minor bug in doco.  https://github.com/Azure/ipam/tree/main/docs/deployment
need to swap the titles 
Override default resource names with custom resource names and deploy as an Azure Function:

with 
Override default resource names with custom resource names and use a private Container Registry:

![image](https://github.com/Azure/ipam/assets/7728453/109c0ad1-4659-44f4-a95f-d317abd07a33)

**Comment by omerzubair** [comment]
hi @DCMattyG ,

Also can you please elaborate what -privateACR does? 
I thought it will make it internal instead of open "all networks"?

**Comment by DCMattyG** [comment]
Thanks @omerzubair, I'll get that typo fixed right away.

As notated in the [deployment guide](https://azure.github.io/ipam/#/deployment/README?id=full-deployment), the `-PrivateACR` switch will create a Private Azure Container Registry, push the containers to that ACR, and use those for your App Service running Azure IPAM. This is for customers whom do not want to use the public images I have hosted and maintain stricter control over those containers being updated.

I hope that answered your question!

**Reply by omerzubair** [reply]
thanks for explaining. I am new to docker . i think this is what i understood. please correct me. 

so it will not use your hosted images. rather it will create container registry resource in my Resource group and upload image to them so i self-reference rather than using your hosted one? 

is there a way to use any scanning of your code before i create the image. i mean reason of this question is that in future if i get latest code drop then my security team will request me to scan code before creating an image.

appreciate your prompt replies.

**Reply by DCMattyG** [reply]
Your understanding is correct.

You can enable the scanning of containers within your ACR using [Defender for Cloud](https://learn.microsoft.com/en-us/azure/defender-for-cloud/agentless-container-registry-vulnerability-assessment) and turn on the vulnerability assessment. This is the same technology that we're using for our publicly hosted containers as well.
